### ISSUE TITLE
I'm confused about the implementation of TMWF, logits.mean(1) may not be the original purpose of TMWF.

### ISSUE THREAD

**Issue by weiyuhao2021** [Xinhao-Deng/Website-Fingerprinting-Library#6]
As far as I know, TMWF is based on object detection model DETR, which contains k (n+1)-classification.
k denotes the number of tab queries, n denotes the number of classes.
Is it appropriate to modify it to one n multi-label classification by average pooling?
据我所知，TMWF基于目标检测模型DETR，需要执行k个n+1分类，k个单标签多分类，然后计算匈牙利匹配
现在简单的将logits平均池化来适配multi-label的数据格式，这样是否会造成性能损失？

**Comment by Xinhao-Deng** [comment]
Sorry for the confusion. We modified the original TMWF because our dataset labels are one-hot vectors, which do not encode the order of website visits. Consequently, we cannot use the current dataset to train the original TMWF.  

In our paper, the goal of multi-tab WF attacks is to identify all websites a user has visited, without considering the visit order. The modified TMWF eliminates the need for prior knowledge of the maximum number of tabs.  

Additionally, in our small-scale study, the TMWF model in WFlib achieved performance comparable to the original version. However, we only used a limited amount of 2-tab traffic, which is insufficient and unconvincing. This is because collecting traffic for different visit orders is highly time-consuming—specifically, the number of possible visit sequences for $N$ websites is $N!$.  

If you have any new findings, please let us know. Thank you!
